packages/editor-ui/src/components/CredentialEdit/CredentialEdit.vue

```diff
@@ -494,6 +491,26 @@ export default defineComponent({
 			this.credentialName = await this.credentialsStore.getNewCredentialName({
 				credentialTypeName: this.defaultCredentialTypeName,
 			});
+
+			const { currentProject, personalProject } = this.projectsStore;
+			const scopes = currentProject?.scopes ?? [
```

Comment:
The `currentProject` should always exist right? Or is it possibly undefined for the personal project at any point? If it always exists, we might throw if undefined rather than assuming default values wdyt?

Reply:
Updated to use also `personalProject` scopes